plugins/fields/list/src/Extension/ListPlugin.php

```diff
@@ -44,9 +44,20 @@ public function onCustomFieldsBeforePrepareField($context, $item, $field)
             return;
         }
 
-        $options = $this->getOptionsFromField($field);
+        if (empty($field->value)) {
+            return;
+        }
 
-        $field->apivalue = [$field->value => $options[$field->value]];
+        $options = $this->getOptionsFromField($field);
+        
+        //API
```

Comment:
```suggestion
        $options = $this->getOptionsFromField($field);

        // API
```
Remove spaces from empty line to fix failing PHPCS check. In addition fix comment style (space after "//").